conf/application.conf

```diff
@@ -51,6 +51,12 @@ controllers {
   }
 }
 
+session{
+  timeoutSeconds = 900
+  countdownInSeconds = 120
+  timeoutUrl = "http://localhost:9379/trader-services/timedout"
```

Comment:
`timeoutUrl` is our own URL and there is no need to configure it via AppConfig, pls use `routes.SessionController.showTimeoutPage` directly in the layout wrappers